xapian-applications/omega/docs/omegascript.rst

```diff
@@ -542,6 +542,15 @@ $set{OPT,VALUE}
 	Similarly, ``XFOO:stemmer`` specifies the stemmer to use for field
 	``XFOO``, with ``stemmer`` providing a default.
 
+$seterror{error_msg}
+	set error message for the current execution, which can also be looked
+	up using ``$error``.
+
+	Using ``$seterror`` error early in template prevent running query.
+	Saves us from executing query unnecessarily.
```

Comment:
This really just says the same thing as the previous sentence, but less clearly - I'd drop it, or rephrase if it's supposed to make a different point.